### PR TITLE
vcf2hapmap

### PR DESCRIPTION
**Pull request by avengerpb**
* [v] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [v] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/vcf2hapmap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+requirements:
+  run:
+    - perl
+
+package:
+  name: vcf2hapmap
+  version: 1.0.0
+
+source:
+  url: https://github.com/avengerpb/vcf2hapmap_conda/archive/1.0.tar.gz
```
Comment (bgruening) [inline recipes/vcf2hapmap/meta.yaml]: Mh, why not using https://github.com/avengerpb/vcf2hapmap directly?

Comment (avengerpb) [inline recipes/vcf2hapmap/meta.yaml]: Hi, I want my recipe for this tool can be accessed more easily and more secured.

File: recipes/vcf2hapmap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  number: 0
+  skip: False
+
+requirements:
+  run:
+    - perl
+
+source:
+  url: https://github.com/avengerpb/vcf2hapmap_conda/archive/1.1.tar.gz
+  md5: 4d4db8b562bc90ee3f150d98d5e510e1
```
Comment (peterjc) [inline recipes/vcf2hapmap/meta.yaml]: I would recommend instead:

```
  sha256: 0fabf645f6e8252529468b846cf5620cc715d251795b4bd5256fe8002f48f927
```

File: recipes/vcf2hapmap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/vcf2hapmap_conda
+  license: Artistic-2.0
```
Comment (peterjc) [inline recipes/vcf2hapmap/meta.yaml]: Also include:

```
   license_file: LICENSE
```

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**